src/pyJiraCli/file_handler.py

```diff
@@ -88,6 +88,10 @@ def set_filepath(self, path:str) -> Ret.CODE:
         formatted_path = os.path.join(*path_comps)
         parent_path = os.path.join(*path_comps[:-1])
 
+        if os.name != 'nt':
```

Comment:
Doesnt Python handle the use of slashed automatically, when using the forward slash exclusively?

Reply:
Without the path operations it didnt work cross platform